### PR TITLE
tests: blobcache: use unique image name

### PR DESCRIPTION
**Pull request by edsantiago**
Prep work for parallelizing bats tests: when pushing images to
local registry, use a pseudorandom unique name to avoid
possibility of collisions/conflicts with other tests. This
is good practice regardless of whether we run serial or parallel.

Also slight refactor of some duplicate code.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

File: tests/blobcache.bats [hunk 1]
```diff
@@ -66,26 +68,26 @@ function _check_matches() {
...
-		assert "$status" -eq 0 "status of `run rm $HOME/.local/share/containers/cache/blob-info-cache-v1.sqlite` must be 0"
-	else
-		run rm /var/lib/containers/cache/blob-info-cache-v1.sqlite
-		assert "$status" -eq 0 "status of `run rm /var/lib/containers/cache/blob-info-cache-v1.sqlite` must be 0"
-	fi
+		cachedir=$HOME/.local/share
+        fi
+        cachefile=$cachedir/containers/cache/blob-info-cache-v1.sqlite
+        run rm $cachefile
+	assert "$status" -eq 0 "status of `run rm $cachefile` must be 0"
```
Comment (Luap99) [inline tests/blobcache.bats:84]: here as well

**Comment by edsantiago** [comment]
Indentation fixed, and repushed. This particular file uses tab indentation (`^I`).

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**